mod_reforged/hooks/entity/tactical/enemies/spider.nut

```diff
@@ -79,6 +79,9 @@
 	    this.m.BaseProperties.Reach = ::Reforged.Reach.Default.BeastSmall;
 	    this.m.Skills.add(::new("scripts/skills/perks/perk_footwork"));
 		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_strength_in_numbers"));
+		this.m.Skills.add(::MSU.new("scripts/skills/perks/perk_rf_between_the_ribs", function(o) {
+    		o.m.RequiredWeaponType = null;
```

Comment:
Between the Ribs is still using the older `IsForceEnabled` style. So this is just to mark it that before merging this PR I'll have to update that perk (and I'll take that chance to do it for all other relevant ones) to convert them to this new system.